tests/test_config.py

```diff
@@ -295,6 +296,93 @@ def test_sort(global_var):
     lux.config.sort = "descending"
 
 
+def test_ordering(global_var):
+    df = pd.read_csv("lux/data/college.csv")
+    lux.config.topk = 5
+    df._ipython_display_()
+    assert len(df.recommendation["Correlation"]) == 5, "Show top 5"
+    score = float("inf")
+    for vis in df.recommendation["Correlation"]:
+        assert vis.score <= score
+        score = vis.score
+    lux.config.ordering = "alphabetical_by_x"
+    df = pd.read_csv("lux/data/college.csv")
+    string = df.recommendation["Correlation"][0].get_attr_by_channel("x")[0].attribute
+    assert len(df.recommendation["Correlation"]) == 5, "Show top 5"
+    for vis in df.recommendation["Correlation"]:
+        assert vis.get_attr_by_channel("x")[0].attribute <= string
+        string = vis.get_attr_by_channel("x")[0].attribute
+    lux.config.ordering = "alphabetical_by_y"
+    df = pd.read_csv("lux/data/college.csv")
+    string = df.recommendation["Correlation"][0].get_attr_by_channel("y")[0].attribute
+    assert len(df.recommendation["Correlation"]) == 5, "Show top 5"
+    for vis in df.recommendation["Correlation"]:
+        assert vis.get_attr_by_channel("y")[0].attribute <= string
+        string = vis.get_attr_by_channel("y")[0].attribute
+    lux.config.ordering = "interestingness"
+    lux.config.topk = 15
+
+
+def test_custom_ordering(global_var):
+    lux.config.topk = 5
+    lux.config.sort = "ascending"
+
+    def sort_by_multiple(collection, desc):
+        collection.sort(
+            key=lambda x: (
+                x.get_attr_by_channel("x")[0].attribute,
+                x.get_attr_by_channel("y")[0].attribute,
+            ),
+            reverse=lux.config.sort,
+        )
+
+    lux.config.ordering = sort_by_multiple
+    df = pd.read_csv("lux/data/college.csv")
+    df._ipython_display_()
+    cmp = (
+        df.recommendation["Correlation"][0].get_attr_by_channel("x")[0].attribute,
+        df.recommendation["Correlation"][0].get_attr_by_channel("y")[0].attribute,
+    )
+
+    for vis in df.recommendation["Correlation"]:
+        assert (
```

Comment:
For every assert, we should add the error message to be thrown if the assertion does not pass.